flake.nix

```diff
@@ -160,21 +160,34 @@
             };
           });
 
-          nix =
-            let
-              officialRelease = false;
-              versionSuffix =
-                if officialRelease
-                then ""
-                else "pre${builtins.substring 0 8 (self.lastModifiedDate or self.lastModified or "19700101")}_${self.shortRev or "dirty"}";
+          nix-util = final.callPackage ./src/libutil/package.nix {
+            inherit
+              fileset
+              stdenv
+              officialRelease
+              versionSuffix
```

Comment:
The version suffix is going to cause unnecessary rebuilds for each commit. I think `officialRelease` is enough "entropy" for the library builds.